### ISSUE TITLE
Unhandled promise rejection, null is not an object (v1.6.2, iOS)

### ISSUE THREAD

**Issue by jarredt** [streem/react-native-select-contact#45]
Getting this error on iOS on v1.6.2 when trying to `selectContactEmail` or `selectContactPhone`:
![Screen Shot 2021-06-02 at 12 59 44 PM](https://user-images.githubusercontent.com/16692671/120544964-e6d88580-c3a2-11eb-8cd0-f004ee3c9184.png)

Confirmed it's working OK on v1.5.0. Deleted `ios/Pods` and reinstalled pods just to be sure.

Android seems to be working OK.

**Comment by Bardiamist** [comment]
Same error

```
import { selectContact } from 'react-native-select-contact';

try {
  await selectContact();
} catch (error) {
  console.log(error.message); // Cannot read property 'openContactSelection' of null
}
```

**Comment by Bardiamist** [comment]
@seanadkinson 1.6.2 don't work on iOS

**Comment by seanadkinson** [comment]
Hmmm, that is very strange.  The only commits of any substance in `1.6.x` were [this](https://github.com/streem/react-native-select-contact/commit/33607c2707907355419cc9d46a03eb9a5f2686c3) and [this](https://github.com/streem/react-native-select-contact/commit/39b2ba2c239aa8fce4cbe2f2c295ccae704b5b7a).  I wouldn't expect either to impact iOS in this way.

It sounds like `SelectContact` is simply not available from `NativeModules`, based on the errors you both have posted.  If you `console.log(NativeModules)` on startup, can you confirm `SelectContact` is not there?

Can you think of any reasons the same code would work properly on 1.5 and not 1.6, based on the above changes?  Did you make sure to `pod install` after `yarn`?

We moved away from react-native a long time ago, so we aren't currently using this library anymore, so need to rely on the community to diagnose and provide fixes.  Thanks!

**Comment by Bardiamist** [comment]
>  If you console.log(NativeModules) on startup, can you confirm SelectContact is not there?

```
'use strict';

import {
    Alert,
    ActionSheetIOS,
    NativeModules,
    Platform
} from 'react-native';

console.log({ NativeModules }); // {"NativeModules": {}}

const { SelectContact, ActionSheetAndroid } = NativeModules;
```
NativeModules is empty object

> Can you think of any reasons the same code would work properly on 1.5 and not 1.6, based on the above changes?

I tried [this](https://github.com/react-native-webview/react-native-webview/commit/08b709967f92b5eba28d63ec287e4754b1fbb632#diff-bdda903df431fd3addf7f53617ba228072c105ac0faa81f3de4cda017a6deaee) but not helped. So not sure, maybe you changed deploy process.

**Comment by Bardiamist** [comment]
@seanadkinson I noticed that I tried with wrong podspec file. I edited [RCTSelectContact.podspec](https://github.com/streem/react-native-select-contact/blob/master/RCTSelectContact.podspec) but now react native using [react-native-select-contact.podspe](https://github.com/streem/react-native-select-contact/blob/master/react-native-select-contact.podspec). So I fixed `source_files` path in `react-native-select-contact.podspec`: https://github.com/streem/react-native-select-contact/pull/46

Also maybe need to delete [RCTSelectContact.podspec](https://github.com/streem/react-native-select-contact/blob/master/RCTSelectContact.podspec).

**Comment by seanadkinson** [comment]
Merged #46, and released `1.6.3`.  Can y'all try again to see if the problem is resolved?  Thanks!

**Comment by jarredt** [comment]
Yep, looks like this is working for me now. Thanks @Bardiamist and @seanadkinson!
